Exit fcoemon command if fcoemon daemon is already running.

NitinYewale opened the pull request:
Made the changes accordingly. I was able to test the if loop where `fcoemon` command exists if the `fcoe` daemon is already running.

Please add
Signed-off-by: Laurence Oberman loberman@redhat.com
Suggested by : Grigory Trenin

gonzoleeman commented:
This functionality looks good to me, but I don't like the layout, and I think the message should be a log message instead of a debug log message.

If it's okay with you, I'll modify it and merge it.

NitinYewale commented:
Yes Please. Please do the necessary changes.

Please add
Signed-off-by: Laurence Oberman loberman@redhat.com
Suggested by : Grigory Trenin

gonzoleeman commented:
Okay, I merged your changes, Here's what I changed:

I made your changes a bit smaller: only 4 lines added. Your version re-wrote stuff that didn't need re-writing
The "signed-off-by" tag signed by somebody that has authority over the project, and Laurence is not a maintainer for this project, so I changed it to "Reviewed-by"
The email address for Laurence should have angle brackets around it.
Your "Suggested by" line is all wrong. It would be "Suggested-by: Name <email-address>", but your left off the dash, added an extra space, and didn't put the email address in place, so there's no way to check with him, or tell him his name has been used.

These things were not too hard to change, but you'll get much better response from maintainers if you supply the patch ready to use, since that means all I have to do is hit a button that says "merge this request", instead of the text surgery I had to perform this time.

Apologies if this sounds like a rebuke, but you need to learn these things if you wish to submit patches to the kernel, for example.

And thank you for your fix and your time!

NitinYewale commented:
Many thanks for everything!


Regards,
Nitin Yewale


On Mon, Aug 10, 2020 at 3:08 AM Lee Duncan <notifications@github.com> wrote:

> Okay, I merged your changes, Here's what I changed:
>
> I made your changes a bit smaller: only 4 lines added. Your version
> re-wrote stuff that didn't need re-writing
> The "signed-off-by" tag signed by somebody that has authority over the
> project, and Laurence is not a maintainer for this project, so I changed it
> to "Reviewed-by"
> The email address for Laurence should have angle brackets around it.
> Your "Suggested by" line is all wrong. It would be "Suggested-by: Name ",
> but your left off the dash, added an extra space, and didn't put the email
> address in place, so there's no way to check with him, or tell him his name
> has been used.
>
> These things were not too hard to change, but you'll get much better
> response from maintainers if you supply the patch ready to use, since that
> means all I have to do is hit a button that says "merge this request",
> instead of the text surgery I had to perform this time.
>
> Apologies if this sounds like a rebuke, but you need to learn these things
> if you wish to submit patches to the kernel, for example.
>
> And thank you for your fix and your time!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openSUSE/fcoe-utils/pull/13#issuecomment-671104111>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADNFT6ZXLXRFTBQRMSY5DHDR74JMDANCNFSM4PWLEG6Q>
> .
>
